cedar-drt/fuzz/fuzz_targets/abac_type_directed_shared.rs

```diff
@@ -135,4 +144,28 @@ pub fn fuzz(input: FuzzTargetInput, def_impl: &impl CedarTestImplementation) {
             Vec::<String>::new()
         );
     }
+
+    if let Ok(test_name) = std::env::var("DUMP_TEST_NAME") {
+        // When the corpus is re-parsed, the policy will be given id "policy0".
+        // Recreate the policy set and compute responses here to account for this.
+        let mut policyset = ast::PolicySet::new();
+        let policy = policy.new_id(ast::PolicyID::from_string("policy0"));
+        policyset.add_static(policy).unwrap();
+        let mut responses = Vec::with_capacity(requests.len());
+        for request in requests.iter() {
+            let authorizer = Authorizer::new();
+            let response = authorizer.is_authorized(request.clone(), &policyset, &input.entities);
+            responses.push(response);
+        }
```

Comment:
same here

Reply:
In fact, would it be possible to share some of this code, which is currently duplicated.  (doesn't have to be in this PR)

Reply:
Reformatted in the latest commit. I didn't deal with the code reuse, but you're right that would be cleaner